odoo/addons/base/models/ir_actions.py

```diff
@@ -420,6 +423,31 @@ def _check_recursion(self):
         if not self._check_m2m_recursion('child_ids'):
             raise ValidationError(_('Recursion found in child server actions'))
 
+    def _get_runner(self):
+        multi = True
+        t = type(self)
+        fn = getattr(t, f'_run_action_{self.action_type}_multi', None)\
+          or getattr(t, f'run_action_{self.action_type}_multi', None)
+        if not fn:
+            multi = False
+            fn = getattr(t, f'_run_action_{self.action_type}', None)\
+              or getattr(t, f'run_action_{self.action_type}', None)
+        if fn.startswith('run_action_'):
```

Comment:
Yup, fixed. Though crash was unlikely to happen as it'd happen IFF somebody added a new state without adding the corresponding method (and we'd get an error instead of just... ignoring the thing entirely) (maybe I should at least add a warning for that case though).